GDD.md

```diff
@@ -11,25 +15,50 @@ This runs on Windows, Linux, and in Web.
 
 ## Development Software
 Programming: Godot Engine
+
 Art: 
+
 Music: 
 
 ## Genre
-Singleplayer, [genre2], [genre3]
+Singleplayer Metroidvania Turn-based Tactics
 
 ## Target Audience
 
 
 # Concept
 ## Gameplay Overview
+Navigate the overworld, armed with the Potion Launcher.
+The overworld is made up of multiple rooms. When player enters a new room, the room is shrouded in SHADOW (room is grayscale), and filled with SHADOW creatures.
+The ALCHEMIST must purify the room without hurting the SHADOW CREATURES, by getting to the SUNflowers (colored).
+Upon reaching enough SUNflowers, the ALCHEMIST's SUNSPARK activates, purifying the room and every SHADOW creatures within it.
 
 ## Theme Interpretations
-Shadow: 
-Alchemy: 
+Shadow: SHADOW has descended upon the world. Player attempts to restore the sun to banish the SHADOW.
 
-## Primary Mechanics
+Alchemy: Player is an ALCHEMIST, who collects reagents and brews new potions that interact with the environment to address challenges. 
 
-## Secondary Mechanics
+## Primary Mechanics
+* Turn cycle: Player makes 1 action, all the NPCs make 1 action each
+* Player is armed with potion gun, with a bunch of different options depending on what they unlocked so far
+* Each shadow has a small moveset, and they tell you in advance what they will do (i.e. Into the Breach)
+* Players must get to 3 out of 5 SUNflowers to extract the latent energy of the SUN and purify the room. This restores color to the room and you can now safely explore.
```

Comment:
we could add a TURN TIMER. That is the player must get the flowers within a set amount of turns

Reply:
I think some sort of turn timer is good idea, but maybe make it feel less arbitrary somehow. Maybe shadow creeping in, making the map smaller if you spend too long. Maybe the sunflowers can wither if left alone too long.

But definitely agreed on some sort of turn timer. I'll add a note.